11Lesson/01Homework/css/style.css

```diff
@@ -0,0 +1,269 @@
+body {
+  width: 100%;
+  font-size: 13px;
+  line-height: 24px;
+  font-family: "Open Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
+  color: #666;
+  background-color: inherit;
+  margin: 0 auto;
+  background-size: cover;
+}
+
+.hero {
+  background: linear-gradient( rgba(0, 0, 0, 0.3), rgba(0, 0, 0, 0.3) ), url("../img/wine14.jpg");
+  color: inherit;
+  background-color: inherit;
+  height: 700px;
+  /* Create the parallax scrolling effect */
+  background-attachment: fixed;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.navbar-default {
+  background: none;
+  border: none;
+  border-bottom: 1px solid #ccc;
+}
+
+.navbar-default .navbar-nav > li > a {
+  color: #fff;
+  background-color: inherit;
+}
+
+.navbar-default .navbar-nav > li > a:hover {
+  color: #fff;
+  background-color: inherit;
+}
+
+i {
+  color: #fff;
+  background-color: inherit;
+  padding: 8px;
+}
+
+.white-text {
+  color: #fff;
+  background-color: inherit;
+}
+
+.signature {
+  max-height: 80px;
+}
+
+.left-box {
+  height: 700px;
+  font-size: 16px;
+  font-family: 'Georgia';
+  font-style: italic;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.lead {
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 28px;
+}
+
+.text-normal {
+  font-style: normal;
+  line-height: 32px;
+}
+
+.black-text {
+  color: #222;
+  background-color: inherit;
+  font-size: 24px;
+  line-height: 32px;
+  margin-bottom: 24px;
+}
+
+.wash {
+  background-image: url("../img/wine16.jpg");
+  background-size: cover;
+  height: 700px;
+}
+
+.boots {
+  background: linear-gradient( rgba(0, 0, 0, 0.3), rgba(0, 0, 0, 0.3) ), url("../img/wine13.jpg");
```

Comment:
If you have two times the same color, you don't need a gradient. Background-color works just fine.